source/uwp/Renderer/dll/dll.cpp

```diff
@@ -5,12 +5,13 @@
 #include <windows.h>
 #include <wrl.h>
 #include <wrl\wrappers\corewrappers.h>
+#include <module.g.cpp>
 
 using namespace Microsoft::WRL;
```

Comment:
>WRL [](http://example.com/codeflow?start=27&length=3)

remove WRL here

Reply:
@beervoley can say for sure, but I'm guessing that there's still some extant WRL stuff in use...